demos/projects/ESPRESSIF/esp32/main/azure_iot_freertos_esp32_main.c

```diff
@@ -0,0 +1,280 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdarg.h>
+#include <string.h>
+
+#include "sdkconfig.h"
+#include "esp_event.h"
+#include "esp_wifi.h"
+#include "esp_wifi_default.h"
+#include "esp_err.h"
+#include "esp_netif.h"
+#include "esp_sntp.h"
+#include "esp_log.h"
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "freertos/semphr.h"
+#include "nvs_flash.h"
+
+#define SNTP_SERVER_FQDN "pool.ntp.org"
+
+static const char *TAG = "sample_azureiot";
+
+static bool g_timeInitialized = false;
+
+/*-----------------------------------------------------------*/
+
+extern void vStartDemoTask( void );
+
+/*-----------------------------------------------------------*/
+
+void vLoggingPrintf( const char * pcFormat,
+                     ... )
+{
+    va_list arg;
+
+    va_start( arg, pcFormat );
+    vprintf( pcFormat, arg );
+    va_end( arg );
+}
+
+/*-----------------------------------------------------------*/
+
+uint64_t ullGetUnixTime( void )
+{
+    time_t now = time(NULL);
+
+    if (now == (time_t)(-1))
+    {
+        ESP_LOGE(TAG, "Failed obtaining current time.\r\n");
+    }
+
+    return now;
+}
+
+/*-----------------------------------------------------------*/
```

Comment:
I see that this file uses this inconsistently ... can we remove it, or add after each function like you did in transport_tls_esp32.c

Reply:
Added /*-----------------------------------------------------------*/
after each function as suggested. As a reminder it was kept without because we wanted to maintain the style used by the samples in esp-idf. But we decided not to anymore, so that's ok.